src/crypto/MnemonicError.ts

```diff
@@ -0,0 +1,12 @@
+export class MnemonicError extends Error {
```

Comment:
As we do `BadKeyError` maybe `BadMnemonicError` instead?

Reply:
We went with `BadKeyError` instead of `KeyError` because the latter seemed too generic. I don't really feel the same way about `MnemonicError`, however it's also not exactly a hill I'm willing to die on.